File: 455-assign-cookies/assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        greedy = 0
        sizei = 0
        while(greedy<len(g) and sizei<len(s)):
            if(s[sizei]>=g[greedy]):
                greedy+=1
            sizei+=1
        return greedy

File: 455-assign-cookies/assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(),g.end());
        sort(s.begin(),s.end());
        int greedy = 0;
        int sizei = 0;
        while(greedy<g.size() && sizei<s.size()){
            if(s[sizei]>=g[greedy]){
                greedy++;
            }
            sizei++;
        }
        return greedy;
    }
};